finalProject/fizzbuzz.js

```diff
@@ -0,0 +1,57 @@
+//This version takes any number and puts it into an array format.
+
+function fizzbuzz(j) {
+  var fizzArray = []
+  for(var i = 1 ; i <= j ; i++){
```

Comment:
This works! Check mdn for better code style. whenever params are around I want some space on their outside